packages/components/src/DateTimePickers/DateTime/date-extraction.js

```diff
@@ -423,6 +423,21 @@ function extractPartsFromDateTime(datetime, options) {
 	};
 }
 
+/**
+ * extract date/time part from datetimeformat
+ * @param {string} datetimeFormat
+ * @param {string} type
+ */
+function extractPartsFromDateTimeFormat(datetimeFormat, type) {
```

Comment:
Can you rename that please ?
The problem is that we have other functions named `extractPartsFromXXX()` and they all extract the same thing : date, time, datetime, textInput, errors. But this function does something else, it extract the date or time within a string

Or you can make 2 functions `extractDateFormat()` / `extractTimeFormat()`

Reply:
OK. I've renamed the function as `extractFormatByPart`